beater/mysqlbeat.go

```diff
@@ -158,11 +149,32 @@ func (bt *Mysqlbeat) Stop() {
 
 ///*** mysqlbeat methods ***///
 
+func (bt *Mysqlbeat) beat(b *beat.Beat) {
+	for server, _ := range bt.config.Servers {
+		logp.Info("Starting prcoessing for server %v", server)
+		err := bt.process_server(server)
+		if err != nil {
+			logp.Err("Error occured when processing %v server, got: %v", server, err)
+		} else {
+			logp.Info("Finished for server %v", server)
+		}
+	}
+}
+
 // beat is a function that iterate over the query array, generate and publish events
-func (bt *Mysqlbeat) beat(b *beat.Beat) error {
+func (bt *Mysqlbeat) process_server(server string) error {
```

Comment:
the argument `server` should be something like `serverName`. Generally `server` in this code refers to a collection of attributes.